docs/content/quickstart/build-bundle.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Build your Bundle
+description: Building your first bundle
+---
+
+Once you have a [porter.yaml](porter-yaml) manifest, it's time to build your first bundle. 
+
+The `porter build` command generates this bundle:
+
+```
+$ porter build
+Copying porter runtime ===>
+Copying mixins ===>
+Copying mixin exec ===>
+
+Generating Dockerfile =======>
+
+Writing Dockerfile =======>
+
+Starting Invocation Image Build =======>
+```
+
+Let's look at these steps in more detail. 
+
+### First, Porter will copy its runtime plus any mixins into the `.cnab/app` directory of your bundle. If you look in the .cnab directory, you'll see something like this 
```

Comment:
Was this intended to be a section heading? The way it renders it looks off. Perhaps we could give it a section title like

```
### Copy Porter Runtime
### Copy Mixins
### Copy Bundle Assets
...
```